sketcherMinimizerFragment.cpp

```diff
@@ -409,7 +409,7 @@ unsigned int sketcherMinimizerFragment::countHeavyAtoms() const
 {
     int n = 0;
     for (auto m_atom : m_atoms) {
-        if (m_atom->atomicNumber != 6) {
+        if (m_atom->atomicNumber != 1) {
```

Comment:
great catch... but I think the right thing to do here is to rename the function to countHeteroatoms. This is used in deciding priorities between fragments and indeed its intended purpose is to count heteroatoms, not heavy atoms